db/migrate/20170309125642_add_alert_ems_ref.rb

```diff
@@ -0,0 +1,6 @@
+class AddAlertEmsRef < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_alert_statuses, :ems_ref, :string
+    add_index :miq_alert_statuses, :ems_ref
```

Comment:
@kbrock I'm pretty sure this isn't the index I need, here is the query using it:
https://github.com/ManageIQ/manageiq/pull/14300/files#diff-5207b5487aa39adaa85b87eb193abb35R215

An explain of that query:
```
                                                                                                                        QUERY PLAN                                                                                 
                                        
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
----------------------------------------
 Limit  (cost=0.00..1.04 rows=1 width=239)
   ->  Seq Scan on miq_alert_statuses  (cost=0.00..1.04 rows=1 width=239)
         Filter: ((miq_alert_id = '1000000000028'::bigint) AND ((resource_type)::text = 'ContainerNode'::text) AND (resource_id = '1000000000001'::bigint) AND ((ems_ref)::text = 'ops-example-1489487734309-58f1aa
1c-dafe-4285-bbae-ec7a788de50d'::text))
(3 rows)
```